circle.yml

```diff
@@ -7,7 +7,7 @@ machine:
 dependencies:
     override:
         - echo y | android update sdk --no-ui --all --filter tools,platform-tools,build-tools-25.0.0,android-22,extra-google-m2repository,extra-google-google_play_services,extra-android-support
-        - ./gradlew dependencies -x lint
+        - ANDROID_HOME=/usr/local/android-sdk-linux ./gradlew dependencies
```

Comment:
This is the config file for CircleCI so it's being run on their (Linux) infrastructure. It doesn't affect your local run.